Pythonwin/pywin/scintilla/scintillacon.py

```diff
@@ -1,23 +1,5 @@
 # Generated by h2py from Include\scintilla.h
```

Comment:
I'd be curious in how to regenerate this file. It could give more insights on why there's undefined names here.

Reply:
There used to be a "h2py.py" file which came with Python - I've no idea if it still does, or if not, what happened to it.

Reply:
Alright so, long story short: These "undefined names" in files generated by h2py have always been like this (meaning broken and unusable) so removing them doesn't hurt.

They are generated like that because h2py does string manipulation on c header files, then tries to evaluate the statements. Methods can successfully be defined in python even if the symbol doesn't exists (lazy evalution), so these get added to the generated python file.

---

For posterity: Some files I've been able to regenerate nearly seamlessly (other than the manual stuff added), some are even improved, others are a complete mess.